{
  "abstract": [
    {
      "text": "An issue due to an ",
      "type": "text"
    },
    {
      "code": "Error",
      "type": "codeVoice"
    },
    {
      "text": " being thrown by a test function and caught by",
      "type": "text"
    },
    {
      "text": " ",
      "type": "text"
    },
    {
      "text": "the testing library.",
      "type": "text"
    }
  ],
  "hierarchy": {
    "paths": [
      [
        "doc://com.apple.documentation/documentation/technologies",
        "doc://org.swift.testing/documentation/Testing",
        "doc://org.swift.testing/documentation/Testing/Issue",
        "doc://org.swift.testing/documentation/Testing/Issue/Kind-swift.enum"
      ]
    ]
  },
  "identifier": {
    "interfaceLanguage": "swift",
    "url": "doc://org.swift.testing/documentation/Testing/Issue/Kind-swift.enum/errorCaught(_:)"
  },
  "kind": "symbol",
  "legalNotices": {
    "copyright": "Copyright &copy; 2025 Apple Inc. All rights reserved.",
    "privacyPolicy": "https://www.apple.com/privacy/privacy-policy",
    "termsOfUse": "https://www.apple.com/legal/internet-services/terms/site.html"
  },
  "metadata": {
    "externalID": "s:7Testing5IssueV4KindO11errorCaughtyAEs5Error_pcAEmF",
    "fragments": [
      {
        "kind": "keyword",
        "text": "case"
      },
      {
        "kind": "text",
        "text": " "
      },
      {
        "kind": "identifier",
        "text": "errorCaught"
      },
      {
        "kind": "text",
        "text": "(any "
      },
      {
        "kind": "typeIdentifier",
        "preciseIdentifier": "s:s5ErrorP",
        "text": "Error"
      },
      {
        "kind": "text",
        "text": ")"
      }
    ],
    "modules": [
      {
        "name": "Swift Testing"
      }
    ],
    "platforms": [
      {
        "beta": false,
        "deprecated": false,
        "name": "iOS",
        "unavailable": false
      },
      {
        "beta": false,
        "deprecated": false,
        "name": "iPadOS",
        "unavailable": false
      },
      {
        "beta": false,
        "deprecated": false,
        "name": "Mac Catalyst",
        "unavailable": false
      },
      {
        "beta": false,
        "deprecated": false,
        "name": "macOS",
        "unavailable": false
      },
      {
        "beta": false,
        "deprecated": false,
        "name": "tvOS",
        "unavailable": false
      },
      {
        "beta": false,
        "deprecated": false,
        "name": "visionOS",
        "unavailable": false
      },
      {
        "beta": false,
        "deprecated": false,
        "name": "watchOS",
        "unavailable": false
      },
      {
        "beta": false,
        "deprecated": false,
        "introducedAt": "6.0",
        "name": "Swift",
        "unavailable": false
      },
      {
        "beta": false,
        "deprecated": false,
        "introducedAt": "16.0",
        "name": "Xcode",
        "unavailable": false
      }
    ],
    "role": "symbol",
    "roleHeading": "Case",
    "symbolKind": "case",
    "title": "Issue.Kind.errorCaught(_:)"
  },
  "primaryContentSections": [
    {
      "declarations": [
        {
          "languages": [
            "swift"
          ],
          "platforms": [
            "iOS",
            "iPadOS",
            "macOS",
            "tvOS",
            "visionOS",
            "watchOS"
          ],
          "tokens": [
            {
              "kind": "keyword",
              "text": "indirect"
            },
            {
              "kind": "text",
              "text": " "
            },
            {
              "kind": "keyword",
              "text": "case"
            },
            {
              "kind": "text",
              "text": " "
            },
            {
              "kind": "identifier",
              "text": "errorCaught"
            },
            {
              "kind": "text",
              "text": "(any "
            },
            {
              "identifier": "doc://com.externally.resolved.symbol/s:s5ErrorP",
              "kind": "typeIdentifier",
              "preciseIdentifier": "s:s5ErrorP",
              "text": "Error"
            },
            {
              "kind": "text",
              "text": ")"
            }
          ]
        }
      ],
      "kind": "declarations"
    },
    {
      "kind": "parameters",
      "parameters": [
        {
          "content": [
            {
              "inlineContent": [
                {
                  "text": "The error which was associated with this issue.",
                  "type": "text"
                }
              ],
              "type": "paragraph"
            }
          ],
          "name": "error"
        }
      ]
    }
  ],
  "references": {
    "doc://com.apple.documentation/documentation/technologies": {
      "abstract": [
        {
          "text": "",
          "type": "text"
        }
      ],
      "identifier": "doc://com.apple.documentation/documentation/technologies",
      "kind": "technologies",
      "role": "overview",
      "title": "Technologies",
      "type": "topic",
      "url": "/documentation/technologies"
    },
    "doc://com.externally.resolved.symbol/s:s5ErrorP": {
      "abstract": [
        {
          "text": "A type representing an error value that can be thrown.",
          "type": "text"
        }
      ],
      "fragments": [
        {
          "kind": "keyword",
          "text": "protocol"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "identifier",
          "text": "Error"
        },
        {
          "kind": "text",
          "text": " : "
        },
        {
          "kind": "typeIdentifier",
          "preciseIdentifier": "s:s8SendableP",
          "text": "Sendable"
        }
      ],
      "identifier": "doc://com.externally.resolved.symbol/s:s5ErrorP",
      "kind": "symbol",
      "role": "symbol",
      "title": "Error",
      "type": "topic",
      "url": "/documentation/Swift/Error"
    },
    "doc://org.swift.testing/documentation/Testing": {
      "abstract": [
        {
          "text": "Create and run tests for your Swift packages and Xcode projects.",
          "type": "text"
        }
      ],
      "identifier": "doc://org.swift.testing/documentation/Testing",
      "kind": "symbol",
      "role": "collection",
      "title": "Swift Testing",
      "type": "topic",
      "url": "/documentation/testing"
    },
    "doc://org.swift.testing/documentation/Testing/Issue": {
      "abstract": [
        {
          "text": "A type describing a failure or warning which occurred during a test.",
          "type": "text"
        }
      ],
      "fragments": [
        {
          "kind": "keyword",
          "text": "struct"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "identifier",
          "text": "Issue"
        }
      ],
      "identifier": "doc://org.swift.testing/documentation/Testing/Issue",
      "kind": "symbol",
      "navigatorTitle": [
        {
          "kind": "identifier",
          "text": "Issue"
        }
      ],
      "role": "symbol",
      "title": "Issue",
      "type": "topic",
      "url": "/documentation/testing/issue"
    },
    "doc://org.swift.testing/documentation/Testing/Issue/Kind-swift.enum": {
      "abstract": [
        {
          "text": "Kinds of issues which may be recorded.",
          "type": "text"
        }
      ],
      "fragments": [
        {
          "kind": "keyword",
          "text": "enum"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "identifier",
          "text": "Kind"
        }
      ],
      "identifier": "doc://org.swift.testing/documentation/Testing/Issue/Kind-swift.enum",
      "kind": "symbol",
      "navigatorTitle": [
        {
          "kind": "identifier",
          "text": "Kind"
        }
      ],
      "role": "symbol",
      "title": "Issue.Kind",
      "type": "topic",
      "url": "/documentation/testing/issue/kind-swift.enum"
    },
    "doc://org.swift.testing/documentation/Testing/Issue/Kind-swift.enum/errorCaught(_:)": {
      "abstract": [
        {
          "text": "An issue due to an ",
          "type": "text"
        },
        {
          "code": "Error",
          "type": "codeVoice"
        },
        {
          "text": " being thrown by a test function and caught by",
          "type": "text"
        },
        {
          "text": " ",
          "type": "text"
        },
        {
          "text": "the testing library.",
          "type": "text"
        }
      ],
      "fragments": [
        {
          "kind": "keyword",
          "text": "case"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "identifier",
          "text": "errorCaught"
        },
        {
          "kind": "text",
          "text": "(any "
        },
        {
          "kind": "typeIdentifier",
          "preciseIdentifier": "s:s5ErrorP",
          "text": "Error"
        },
        {
          "kind": "text",
          "text": ")"
        }
      ],
      "identifier": "doc://org.swift.testing/documentation/Testing/Issue/Kind-swift.enum/errorCaught(_:)",
      "kind": "symbol",
      "role": "symbol",
      "title": "Issue.Kind.errorCaught(_:)",
      "type": "topic",
      "url": "/documentation/testing/issue/kind-swift.enum/errorcaught(_:)"
    }
  },
  "schemaVersion": {
    "major": 0,
    "minor": 3,
    "patch": 0
  },
  "sections": [],
  "variants": [
    {
      "paths": [
        "/documentation/testing/issue/kind-swift.enum/errorcaught(_:)"
      ],
      "traits": [
        {
          "interfaceLanguage": "swift"
        }
      ]
    }
  ]
}
